.github/workflows/check-formatting.yml

```
@@ -35,6 +35,11 @@ jobs:
 
       - name: Check formatting with Prettier
         run: bun run format:check
+        continue-on-error: true
+
+      - name: Auto-fix formatting if check fails
+        if: steps.format-check.outcome == 'failure'
```

actionlint (1.7.4)

property "format-check" is not defined in object type {}

(expression)